Trinap/Sources/Presenter/TabBar/Main/PhotographerDetail/CollectionView/PhotographerProfileCell.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  PhotographerProfileCell.swift
+//  Trinap
+//
+//  Created by kimchansoo on 2022/11/29.
+//  Copyright © 2022 Trinap. All rights reserved.
+//
+
+import UIKit
+
+final class PhotographerProfileCell: BaseCollectionViewCell {
+    
+    // MARK: UI
+    lazy var filterView = FilterView(filterMode: .photographer)
+    private lazy var profileImage = ProfileImageView()
+    
+    private lazy var nickNameLabel = UILabel().than {
+        $0.text = "어디로든떠나요"
+        $0.textColor = .black
+        $0.font = TrinapFontFamily.Pretendard.bold.font(size: 20)
+    }
+     
+    private lazy var locationLabel = UILabel().than {
+        $0.text = "서울시 성동구"
+        $0.textColor = TrinapAsset.gray40.color
+        $0.font = TrinapFontFamily.Pretendard.regular.font(size: 14)
+    }
+    
+    // MARK: Properties
+    
+    // MARK: Initializers
+    
+    // MARK: Methods
+    override func configureHierarchy() {
+        self.addSubviews([profileImage, nickNameLabel, locationLabel, filterView])
```

Comment:
collectionViewCell이면 contentView에 addSubview 해주어야 할 것 같습니당
```suggestion
        self.contentView.addSubviews([profileImage, nickNameLabel, locationLabel, filterView])
```